src/web/components/Nav/ExpandedMenu/ExpandedMenu.tsx

```diff
@@ -8,17 +8,23 @@ import { from, until } from '@guardian/src-foundations/mq';
 import { ExpandedMenuToggle } from './ExpandedMenuToggle/ExpandedMenuToggle';
 import { Columns } from './Columns';
 
-const showExpandedMenuStyles = css`
-    ${from.desktop} {
-        display: block;
-        overflow: visible;
-    }
-    ${until.desktop} {
-        transform: translateX(0%);
-    }
-`;
+const mainMenuStyles = (CHECKBOX_ID: string) => css`
+        /*
+        IMPORTANT NOTE:
+        we need to specify the adjacent path to the a (current) tag
+        to apply styles to the nested tabs due to the face we use ~
+        to support NoJS
+    */
+   ${`#${CHECKBOX_ID}:checked ~ & {
+        ${from.desktop} {
+            display: block;
+            overflow: visible;
+        }
+        ${until.desktop} {
+            transform: translateX(0%);
```

Comment:
Comment why?

Reply:
below there is a style `transform: translateX(-110%);` that pushes the Nav off the screen. If the checkbox is `:checked` then it is set to 0% which makes it reappear. We could look at changing this styling logic, but I kept it here as it was not the main focus. Changing this could potentially help us fix the unit test